apis/v1alpha1/generator.yaml

```diff
@@ -4,7 +4,25 @@ ignore:
       - IdentityProvider
       - ResourceServer
       - UserImportJob
-      - UserPool
       - UserPoolClient
       - UserPoolDomain
 model_name: cognito-idp
+
+resources:
+  UserPool:
+    renames:
+      operations:
+        DescribeUserPool:
+          input_fields:
+            UserPoolId: Id
+        UpdateUserPool:
+          input_fields:
+            UserPoolId: Id
+        DeleteUserPool:
+          input_fields:
+            UserPoolId: Id
+        CreateUserPool:
+          input_fields:
+            PoolName: Name
+    tags:
+      ignore: true
```

Comment:
Ideally we need tags for tracking purposes, wanna do this in a seperate PR?

Reply:
i can add it on here...looks like it's a separate api for find and update...

Reply:
added